common-template/src/main/resources/templates/slack/Default/default.md

```diff
@@ -0,0 +1 @@
+{#if data.context.display_name??}<{data.inventory_url.raw}|{data.context.display_name}> triggered {data.events.size()} event{#if data.events.size() > 1}s{/if}{#else}{data.events.size()} event{#if data.events.size() > 1}s{/if} triggered{/if} from {data.source.application.display_name} - {data.source.bundle.display_name}. <{data.application_url.raw}|Open {data.source.application.display_name}>
```

Comment:
Why do these URLs use a safe escape, but the Google Chat and MS Teams ones do not?